source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,228 @@
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass a term as a string to the ``Filters.text()`` method to specify the
+term in your text search. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin termExample
+   :end-before: end termExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+To match multiple terms in your text search, separate each term
+with spaces in the ``Filters.text()`` builder method. When you pass the 
+text search query returned by the builder method to the
+``find()`` method, it returns documents that match any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+terms "Fate" or "7".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin multipleTermExample
+   :end-before: end multipleTermExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a phrase with escaped quotes to the ``Filters.text()`` method to
+specify the phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "Fate of the Furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each term you want to exclude from your text search, prefix the term
+with a minus sign in string you pass to the ``Filters.text()`` builder
+method. 
+
+You must have at least one text search term if you want to
+exclude terms from your search. If you have multiple text indexes, the
+search applies to all indexes. 
```

Comment:
From reading your previous revisions, I thought you can only have one text index. I think the only exception is if you use the Atlas full text search feature which we haven't covered in this document.

Reply:
In the text search section, I have 
> When you execute the ``find()`` method, MongoDB runs a text search on
the fields indexed with the text index on the collection.

I think this covers multiple text indexes as well?

Reply:
I think my comment referred to an older commit:

> If you have multiple text indexes, the
> search applies to all indexes.